contracts/package.json

```diff
@@ -18,7 +18,7 @@
     "prepublishOnly": "pnpm compile && ./scripts/prepublish_generate_abi_folder",
     "publish-beta": "pnpm publish --tag beta",
     "publish-prod": "npm dist-tag add @chainlink/contracts@0.8.0 latest",
-    "solhint": "solhint --max-warnings 593 \"./src/v0.8/**/*.sol\""
+    "solhint": "solhint --max-warnings 502 \"./src/v0.8/**/*.sol\""
```

Comment:
is there any reason for this specific number :) I haven't used this before

Reply:
This is the combined number of issues still in the repo, we want this to exactly match as introducing any new issue would make CI fail. When solving issues, this number should be decreased by the number solved. Eventually this will be 0